ingestion/src/metadata/data_quality/validations/base_test_handler.py

```diff
@@ -87,7 +88,7 @@ def get_test_case_param_value(
             (param.value for param in test_case_param_vals if param.name == name), None
         )
 
-        if not value:
+        if value is None:
```

Comment:
is it expected to move from considering `[]`,`""`, `None` to only considering `None`?

Reply:
retained original behavior and changed how the parameter is consumed